src/workers/jobs/index.js

```diff
@@ -805,6 +810,10 @@ export async function assignTexters(job) {
           .update({ assignment_id: null });
       }
 
+      for (const assignmentId of deletedAssignmentIds) {
+        await trx("assignment").where({ id: assignmentId }).del();
```

Comment:
As I understand it, there's no need to delete assignments. It causes problems with opt_out foreign keys and if a texter gets assigned contacts on a campaign after, they'll have to have the assignment recreated.

Instead, assignments with 0 campaign contacts should just be hidden on the client or filtered before being returned